google/cloud/forseti/common/gcp_type/firewall_rule.py

```diff
@@ -576,13 +604,22 @@ def is_equivalent(self, other):
         Returns:
           bool: comparison result
         """
-        return (self.direction == other.direction and
-                self.network == other.network and
-                self._source_tags == other._source_tags and
-                self._target_tags == other._target_tags and
-                self.source_ranges == other.source_ranges and
-                self.destination_ranges == other.destination_ranges and
-                self.firewall_action.is_equivalent(other.firewall_action))
+        direction = self.direction == other.direction
+        network = self.network == other.network
+        source_tags = self._source_tags == other._source_tags
+        target_tags = self._target_tags == other._target_tags
+        source_ranges = self.source_ranges == other.source_ranges
+        destination_ranges = self.destination_ranges == other.destination_ranges
+        firewall_action = (self.firewall_action.
```

Comment:
This would not need to wrap. There is no separate states.
```
self.firewall_action.is_equivalent(other.firewall_action))
```

Reply:
It's exceeding the maximum characters limit of the line so had to break it down.

Reply:
Ah, if that is the case, wrap like this:
```
firewall_action = (
    self.firewall_action.is_equivalent(other.firewall_action))
```